listenbrainz_spark/__init__.py

```diff
@@ -21,8 +25,5 @@ def init_spark_session(app_name):
         context.setLogLevel("ERROR")
         sql_context = SQLContext(context)
     except AttributeError as err:
-        raise AttributeError('Cannot initialize Spark session "{}": {} \n {}.'.format(app_name, type(err).__name__,
-            str(err)))
-    except Exception as err:
-        raise Exception('An error occurred while initializing Spark session "{}": {} \n {}.'.format(app_name,
-            type(err).__name__, str(err)))
+        logging.error('Cannot initialize Spark session "{}": {} \n{}'.format(app_name, type(err).__name__, str(err)))
```

Comment:
@mayhem  if any of the scripts calling init_spark_session fails, the error will be raised in the function itself.  In the calling script, we need not catch the error message, we can just catch the specific exception and display any additional message if we want to, in our case we have just displayed 'aborting'.